ait/core/server/stream.py

```diff
@@ -55,6 +55,7 @@ def process(self, input_data, topic=None):
         """
         Invokes each handler in sequence.
         Publishes final output data.
+        Returns None if a handler cannot process input data
```

Comment:
At the risk of being a bit pedantic, `process()` returns None even if it successfully processes input data. I would like to see a bit of elaboration about None causing the handler string to terminate and no data to be published, etc.